.github/workflows/check-latest-release.yml

```
@@ -73,7 +73,10 @@ jobs:
         id: scan-image
         uses: anchore/scan-action@v3
         with:
-          image: buildpacksio/lifecycle:${{ steps.read-go.outputs.latest-release-version }}
+          image: buildpacksio/lifecycle:${{ steps.read-versions.outputs.latest-release-version }}
+          fail-build: true
+          severity-cutoff: high
```

```suggestion
          severity-cutoff: medium
```
(We use the default of `medium` currently...)